app/isomorphic/components/story-templates/blank.m.css

```diff
@@ -0,0 +1,330 @@
+.wrapper {
+  border-bottom: 1px solid var(--brand-primary);
+  margin: 16px auto;
+  padding-bottom: 16px;
+
+  .headline {
+    font-size: var(--base-larger);
+  }
+
+  .qt-image-16x9 {
+    padding-top: 56.25%;
+    margin: 0;
+    position: relative;
+    overflow: hidden;
+    margin-bottom: 16px;
+
+    & img {
+      position: absolute;
+      top: 0;
+      left: 0;
+      bottom: 0;
+      object-fit: contain;
+      height: 100%;
+      width: 100%;
+    }
+  }
+
+  .author {
+    font-size: var(--base);
+    padding: 16px 0;
+    font-style: italic;
+  }
+
+  .story-card {
+    margin-bottom: 16px;
+  }
+
+  a {
+    color: var(--brandPrimary);
+    opacity: 0.8;
+  }
+
+  a:hover {
+    opacity: 1;
+  }
+
+  :global .story-element,
+  :global .story-element-text p {
+    letter-spacing: normal;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    font-size: var(--base-medium);
+    line-height: 1.7;
+    margin-bottom: 8px;
+  }
+
+  :global .story-element-text-also-read {
+    text-align: left;
+    hyphens: auto;
+    a {
+      color: var(--brandPrimary);
+    }
+  }
+
+  :global .story-element-text p {
+    margin-bottom: 12px;
+  }
+
+  :global .story-element-text-question p {
+    font-weight: var(--bold);
+  }
+
+  :global .story-element-text-q-and-a {
+    margin-bottom: 12px;
+
+    :global .question p {
+      font-size: var(--base-medium);
+      font-weight: var(--bold);
+      line-height: 1.67;
+      margin-bottom: 8px;
+    }
+
+    :global .answer {
+      position: relative;
+      padding: 0 12px;
+    }
+
+    :global .answer::before {
+      content: "";
+      border-left: 2px solid var(--brand-primary-dark);
+      height: 100%;
+      position: absolute;
+      top: 0;
+      left: 0;
+    }
+  }
+
+  ul,
+  ol {
+    margin-left: 16px;
```

Comment:
this won't affect the tags used globally right ?

Reply:
inside the styleName.